DFC.ServiceTaxonomy.Editor/Recipes/Skillstoolkit.Recipe.json

```diff
@@ -0,0 +1,431 @@
+{
+  "name": "",
+  "displayName": "",
+  "description": "",
+  "author": "",
+  "website": "",
+  "version": "",
+  "issetuprecipe": false,
+  "categories": [],
+  "tags": [],
+  "steps": [
+    {
+      "name": "content",
```

Comment:
We should always use `ContentNoCache`, rather than `content` (unless we need to inject C#, then we'd use `CSharpContent`).